### ISSUE TITLE
Remove "Instance settings" section?

### ISSUE THREAD

**Issue by joe-cole1** [lowercasename/gathio#168]
I deployed gathio via Docker Compose. Very happy with how it works, but I dislike the Instance Description and Instance Settings block on the front page.

I've removed the Instance Description by SSH'ing into the container and editing instance-description.md, but this manual editing does not survive a container recreate and does not help with the Instance Settings block.

Am I missing something? If not, can I submit this as a feature request to control the visibility of these two sections via the config file?

Thank you!

![Screenshot 2024-08-02 182044](https://github.com/user-attachments/assets/ebe83e70-c616-4cfe-a673-212fbc2dcf72)

**Comment by lowercasename** [comment]
I have no problem with adding some config to remove the instance description - I'll see if I can get to it this week.

The fact that your instance description and custom pages don't survive a container recreate is a bug, though! They need to live outside the container, of course.

**Comment by joe-cole1** [comment]
Thanks for the response! Really loving the platform so far. 

I did not edit them outside the container, so I didn't expect them to survive. 

Would I simply create a static folder, much like the config and images folder, then map static in the docker compose?

**Comment by lowercasename** [comment]
Yep, that would do it! I just need to make it the default in the docker compose file and in the docs.

**Comment by lowercasename** [comment]
I've improved the docs around the Docker installation and made that external static folder the default in the docs in this PR: https://github.com/lowercasename/gathio/pull/189/files. 

Still need to get around to making the instance settings block hideable/more configurable.
